pkgs/applications/misc/rofi-file-browser/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "rofi-file-browser-extended";
-  version = "1.1.1";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "marvinkreis";
     repo = pname;
-    rev = version;
-    sha256 = "10wk5sif3bmvsgyk2gdy0qhpv1b37zgzf89n3h0yh7pg195fi2gn";
+    rev = "35d4a2b7bd0fa4a42f5d5a791c4bf0a0da65a974";
```

Comment:
Why are we no longer using the tag?

Reply:
I copy-pasted the result of: `nix-prefetch-github-latest-release --nix --fetch-submodules marvinkreis rofi-file-browser-extended`

Reply:
I copy-pasted the result of: `nix-prefetch-github-latest-release --nix --fetch-submodules marvinkreis rofi-file-browser-extended`

Reply:
Please revert the change. We want to keep using the tag. In the future you can use nix-update to make this easier.